src/main/java/com/unzer/payment/Unzer.java

```diff
@@ -800,6 +803,17 @@ public Charge chargeAuthorization(String paymentId, BigDecimal amount, String pa
         return paymentService.chargeAuthorization(paymentId, amount, paymentReference);
     }
 
+    /**
+     * Charge (Capture) with custom Charge parameters
+     *
+     * @param charge used for the charge of an authorization. Charge.paymentId must be not null.
+     * @return Charge with id
+     * @throws HttpCommunicationException in case communication to Unzer didn't work
+     */
+    public Charge chargeAuthorization(Charge charge) throws HttpCommunicationException {
```

Comment:
I would have the paymentId as a parameter similar to the other methods since this is simply the payment reference. like (String paymentId, Charge charge).
paymentId will always be needed for this request. All other properties of the charge are optional.